app/models/concerns/hud_reports/start_to_move_in_question.rb

```diff
@@ -36,7 +36,7 @@ def start_to_move_in_question(question:, members:, populations: sub_populations)
             when :average
               value = 0
               scope = relevant_members.where(col_cond).where(a_t[:move_in_date].between(@report.start_date..@report.end_date))
-              stay_lengths = scope.pluck(a_t[:time_to_move_in])
+              stay_lengths = scope.pluck(a_t[:time_to_move_in]).compact
```

Comment:
I don't think we want to compact these since we count them to get averages on the next line.  Probably we need to coalesce with 0, which I thought was what `.sum(0.0)` was doing, guess not.  How about?
```suggestion
              stay_lengths = scope.pluck(a_t[:time_to_move_in]).map { |len| len.precence || 0 }
```

Reply:
alternatively if we have ArelHelper already:
```suggestion
              stay_lengths = scope.pluck(cl(a_t[:time_to_move_in], 0))
```

Reply:
Thanks! sounds like it's okay to treat nulls as 0 in this case

Reply:
```suggestion
              # treat null values as having 0 time in this case
              stay_lengths = scope.pluck(a_t[:time_to_move_in]).map { |len| len || 0 }
```